colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -113,20 +113,20 @@ with ClientConnectionHandler with Sender[P, Callback] with ManualUnbindHandler {
 
   import colossus.core.WorkerCommand._
   import config._
-  import context.worker.metrics
+  implicit val namespace = context.worker.system.metrics / config.name
```

Comment:
This actually brings up a question, which I"m not sure I know if we answered or not, which is , how should Client addresses be namespace?  I guess IOSystem?  I can't think of anything lower that would make sense